cpu/cc2538/radio/cc2538_rf_radio_ops.c

```diff
@@ -91,6 +92,7 @@ static int _request_transmit(ieee802154_dev_t *dev)
 {
     (void) dev;
 
+    cc2538_tx_busy = true;
     if (cc2538_csma_ca_retries < 0) {
```

Comment:
shouldn't this only be set in the `else` branch?

Reply:
In both cases the TX Chain will be busy. Therefore, this should stay there